content/docs/0.8.x/integrations/sli_provider/index.md

```diff
@@ -18,69 +18,133 @@ indicators:
 
 **Note:** This SLI configuration file will then be stored in Keptn's configuration store using the [keptn add-resource](../../reference/cli/commands/keptn_add-resource) command.
 
+The [Keptn CloudEvents](#cloudevents) an *SLI-provider* has to subscribe to is:
+
+- `sh.keptn.event.get-sli.triggered`
+
+## Write your custom SLI-provider
+
+Like a Keptn-service, an SLI-provider has the following characteristics: 
+
+* has a **subscription** to an event (i.e., `sh.keptn.event.get-sli.triggered`)
+* sends a **started event** to inform Keptn about receiving the event and acting on it
+* processes functionality and can therefore leverage additional tools, e.g., through their REST interface
+* sends a **finished event** to inform Keptn about its execution status and the result
+
+### Subscription to Keptn event
+
+An *SLI-provider* starts working, when receiving a Keptn CloudEvent of type:
+
+- [sh.keptn.event.get-sli.triggered](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#get-sli) 
```

Comment:
we will need to update the link to the spec once we have released the spec containing the Keptn 0.8 Events

Reply:
True. Therefore we have the issue (https://github.com/keptn/keptn.github.io/issues/667) that addresses this when the spec is online.